completions/doas

```diff
@@ -0,0 +1,45 @@
+# doas(1) completion                                       -*- shell-script -*-
+
+_comp_cmd_doas()
+{
+    local cur prev words cword split
+    _init_completion -s || return
+
+    local i
+
+    for ((i = 1; i <= cword; i++)); do
+        if [[ ${words[i]} != -* ]]; then
+            local PATH=$PATH:/sbin:/usr/sbin:/usr/local/sbin
+            local root_command=${words[i]}
+            _command_offset $i
+            return
+        fi
+        [[ ${words[i]} == -@(!(-*)[uCLs]) ]] &&
+            ((i++))
+    done
+
+    case "$prev" in
+        -!(-*)u)
```

Comment:
Same as above; I simply use `-u` and `-C` in my personal doas completion file.

Reply:
That will fail to match the case where `u` is the last in a set of bundled short options, like `doas -nu <TAB>`. Similary above.